### PR TITLE
Update Playground dependencies in package.json

### PR DESCRIPTION
**Pull request by adamziel**
## What?

Updates the versions of WordPress Playground packages and makes the necessary adjustments to make the tests work with these new version. WordPress Playground `0.5.4` is compatible with Node.js v18 via polyfills, which means wp-now will now be able to use Blueprint steps like `installPlugin`.

## Testing Instructions

Confirm the CI checks pass

cc @sejas @danielbachhuber  @wojtekn

### CODE REVIEW

File: packages/wp-now/src/wp-now.ts [hunk 1]
```diff
@@ -50,19 +50,6 @@ export default async function startWPNow(
 		requestHandler: {
 			documentRoot,
 			absoluteUrl: options.absoluteUrl,
-			isStaticFilePath: (path) => {
```
Comment (adamziel) [inline packages/wp-now/src/wp-now.ts:53]: This is now handled by Playground core.

**Comment by sejas** [comment]
Awesome! Thanks for creating the PR 🙏 .
The code looks good.  I'm testing it locally right now.
Can we update also https://github.com/WordPress/playground-tools/blob/trunk/.nvmrc ?

```diff
- v18.13.0
+ v18.18.2
```

When trying to install the dependencies I got this error:

```
❯ nvm use                                                                                                                                      8s 14:29:18
Found '/Users/macbookpro/proyectos/a8c/playground-tools/.nvmrc' with version <v18.13.0>
Now using node v18.13.0 (npm v8.19.3)
❯ npm install                                                                                                                                     14:29:22
npm ERR! code EBADENGINE
npm ERR! engine Unsupported engine
npm ERR! engine Not compatible with your version of node/npm: @php-wasm/node@0.5.4
npm ERR! notsup Not compatible with your version of node/npm: @php-wasm/node@0.5.4
npm ERR! notsup Required: {"node":">=18.18.2","npm":">=8.11.0"}
npm ERR! notsup Actual:   {"npm":"8.19.3","node":"v18.13.0"}

npm ERR! A complete log of this run can be found in:
npm ERR!     /Users/macbookpro/.npm/_logs/2024-01-11T14_29_25_331Z-debug-0.log
```

**Review by sejas — APPROVED** [review]
I tested it locally and it works great.
- I think we can follow Github's recommendation to delete [`seemsLikeAPHPFile`](https://github.com/WordPress/playground-tools/pull/139/files#diff-eeed5d391a89564a5ff176b28d0ab5a31e01b94fb75f4ccc1d67f80297586595L35) function.
- Let's update `.nvmrc` to use the same `@php-wasm/node` node version requirement:`v18.18.2`.

### Testing steps:

- Update `.nvmrc` to use `v18.18.2`
- Run: 
```
nvm use
npm install
npx nx build wp-now
node dist/packages/wp-now/cli.js start 
```
- Observe `wp-now` successfully started.

**Comment by sejas** [comment]
@adamziel, I applied these changes to this PR:

> I think we can follow Github's recommendation to delete seemsLikeAPHPFile function.

https://github.com/WordPress/playground-tools/pull/139/commits/068db66f415536b4ca31eb1fd0412d16df3fb043

> Let's update .nvmrc to use the same @php-wasm/node node version requirement:v18.18.2.

https://github.com/WordPress/playground-tools/pull/139/commits/39161f667a0844fd873208b5786940622cd91b1b

**Comment by adamziel** [comment]
Awesome, thank you so much @sejas! Let's ship! 🚢 🚢 🚢

**Pull request merged**